ci/pipelines/shared/jinja.variables.yml

```diff
@@ -202,3 +202,15 @@ tests:
   PLATFORM: windows
   RAM: '64'
   name: WindowsUnit
+- ARTIFACT_SLUG: apicheck
+  CALL_STACK_TIMEOUT: '20700'
```

Comment:
maybe not applicable to this job? But in general CALL_STACK_TIMEOUT should be 10-15 minutes less than EXECUTE_TEST_TIMEOUT, which you've set to 1h.  So `2700` might be a better choice.